src/documentation/pages/components/price/examples/_base.vue

```diff
@@ -0,0 +1,49 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" padding="-p--0" :tabs="exampleTabs">
```

Comment:
```suggestion
  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```

Reply:
I think we can update it to -p--3 to match with the current padding in production.

Reply:
-p--3 is the padding marked by default in the new doc system, so it's not neccessary to include it